appveyor.yml

```diff
@@ -0,0 +1,19 @@
+install:
+  - cinst -y jdk7
+  - cinst -y ant -ignoreDependencies
+  - set JAVA_HOME=C:\Program Files\Java\jdk1.7.0
+  - set PATH=%JAVA_HOME%\bin;%PATH%
+  - set BUCK_EXTRA_JAVA_ARGS=-Djna.nosys=true
+  - python --version
+  - ant -version
+
+
+build_script:
+  - "bin\\buck build buck"
+
+test_script:
+  - "bin\\buck test --all --test-selectors=@windows_failures.txt"
+
+artifacts:
+  - path: buck-out\gen\programs\buck.pex
```

Comment:
I actually made the changes recently to allow Buck to be built on Linux and be usable on Windows (and I suspect OS X).  I didn't test the other way around, however!